scripts/population/load-regions

```diff
@@ -1,11 +1,12 @@
-#! /bin/bash
+#!/bin/bash
 set -euo pipefail
 
 source $(dirname "${BASH_SOURCE}")/env-vars
 export PGPASSWORD=$POSTGRES_PASSWORD;
 
 COUNTRY=$1
-LEVELS=${2:-2,4}
+#LEVELS=${2:-2,4}
+LEVELS=(0,1)
```

Comment:
Are these GADM levels? Are the admin levels imported into the table different than before? If so, there are a couple of places throughout the code that assume certain values for admin levels.

Reply:
Yes, gadm call it 0 and 1.